theClanless/src/main/java/theClanless/cards/ShatteringBlow.java

```diff
@@ -0,0 +1,83 @@
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.localization.CardStrings;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import theClanless.characters.TheClanless;
+import theClanless.theClanlessMod;
+
+import static theClanless.theClanlessMod.makeCardPath;
+
+public class ShatteringBlow extends AbstractDynamicCard {
+
+    // public static final String ID = DefaultMod.makeID(${NAME}.class.getSimpleName()); // USE THIS ONE FOR THE TEMPLATE;
+    public static final String ID = theClanlessMod.makeID("ShatteringBlow"); // DELETE THIS ONE.
+    public static final String IMG = makeCardPath("ShatteringBlow.png");// "public static final String IMG = makeCardPath("${NAME}.png");
+    // This does mean that you will need to have an image with the same NAME as the card in your image folder for it to run correctly.
+
+
+    // /TEXT DECLARATION/
+
+
+    // STAT DECLARATION
+    private static final CardStrings cardStrings = CardCrawlGame.languagePack.getCardStrings(ID);
+    public static final String DESCRIPTION = cardStrings.DESCRIPTION;
+    public static final String UPGRADE_DESCRIPTION = cardStrings.UPGRADE_DESCRIPTION;
+
+    private static final CardRarity RARITY = CardRarity.UNCOMMON; //  Up to you, I like auto-complete on these
+    private static final CardTarget TARGET = CardTarget.ENEMY;  //   since they don't change much.
+    private static final CardType TYPE = CardType.ATTACK;       //
+    public static final CardColor COLOR = TheClanless.Enums.POTENCE;
+
+    private static final int COST = 1;  // COST = ${COST}
+    private static final int UPGRADED_COST = 1; // UPGRADED_COST = ${UPGRADED_COST}
+
+    private static final int DAMAGE = 8;    // DAMAGE = ${DAMAGE}
+    private static final int UPGRADE_PLUS_DMG = 3;  // UPGRADE_PLUS_DMG = ${UPGRADED_DAMAGE_INCREASE}
+
+    // /STAT DECLARATION/
+
+
+    public ShatteringBlow() { // public ${NAME}() - This one and the one right under the imports are the most important ones, don't forget them
+        super(ID, IMG, COST, TYPE, COLOR, RARITY, TARGET);
+        this.damage = this.baseDamage = DAMAGE;
+
+    }
+
+
+    // Actions the card should do.
+    @Override
+    public void use(AbstractPlayer p, AbstractMonster m) {
+        if (damage >= m.currentBlock) {
+            AbstractDungeon.actionManager.addToBottom(
+                    new RemoveAllBlockAction(m, p)
+            );
+            AbstractDungeon.actionManager.addToBottom(
+                    new DamageAction(m, new DamageInfo(p, damage, damageTypeForTurn), AbstractGameAction.AttackEffect.SLASH_HORIZONTAL)
+            );
+        } else {
+            AbstractDungeon.actionManager.addToBottom(
+                    new DamageAction(m, new DamageInfo(p, damage * 2, damageTypeForTurn), AbstractGameAction.AttackEffect.SLASH_HORIZONTAL)
```

Comment:
Should this multiplier be a variable? "Does double/triple damage to armor"

Reply:
that's not a bad idea, it's not often that you would need triple damage to block but there are a couple of monsters that it would help